workers-site/index.js

```diff
@@ -61,7 +63,7 @@ function notFoundAsset(assetName = '404.html') {
     const notFoundURL = new URL(request.url)
     notFoundURL.pathname = assetName
 
-    return new Request(notFoundURL, { status: 404 })
```

Comment:
missed that the request had `status: 404` before **d'oh**

Reply:
lul right? that was the facepalm.